source/en/skins-system.rst

```diff
@@ -0,0 +1,163 @@
+        ]
+      }
+
+   By default textures proxying isn't used for this query. To enable it, add an additional GET parameter
+   :samp:`?proxy=true`.
+
+   The server will return an empty response with :samp:`204` status in case when textures aren't found.
+
+------------------------------------------------------------------------------------------------------------------------
+
+You can also pass a number of additional GET parameters when making any of the above requests. They will be used
+to analyze the usage of the service by different versions of the game.
+
+:version: The version of the protocol that the skins are being requested. This is currently version :samp:`2`,
+          i.e. you need to specify :samp:`version=2`.
+
+:minecraft_version: The version of Minecraft that the request is made from.
+
+:authlib_version: The version of the Authlib used. This option is relevant for Minecraft versions 1.7.6+,
+                  where a separate library is used to load skins rather than an in-game code.
+
+An example of a textures query with passing of parameters described above:
+
+.. code-block:: text
+
+   http://skinsystem.ely.by/textures/erickskrauch?version=2&minecraft_version=1.14.0&authlib_version=1.5.25
+
+Additional URLs
++++++++++++++++
+
+You can also perform a skin and cape request by passing the nickname through the GET parameter. This feature is used
+to pass analytical parameters on the game versions up to 1.5.2, when the nickname is simply appended to the end of the
+line. To do this, the entire string is arranged in such a way that the last parameter was :samp:`name`, after appending
+a nickname to which you get a full request string for textures.
+
+.. function:: /skins?name={nickname}.png
+
+   See the `skin request <#skin-request>`_.
+
+.. function:: /cloaks?name={nickname}.png
+
+   See the `cape request <#cape-request>`_.
+
+Example of requests for textures with parameters above:
+
+.. code-block:: text
+
+   http://skinsystem.ely.by/skins?version=2&minecraft_version=1.5.2&name=erickskrauch.png
+   http://skinsystem.ely.by/cloaks?version=2&minecraft_version=1.4.7&name=notch
+
+.. _textures-proxy:
+
+Textures proxying
+=================
+
+Ely.by's service of skins system obtains textures from the official skin system in case no information about textures
+for the requested user name was found in the database. The request will also be proxied if a skin entry is found,
+but it's default.
+
+To improve the throughput of the proxying algorithm, information about textures is cached in the 2 stages:
+
+* Player's names and UUIDs matches are stored
+  `for 30 days <https://help.mojang.com/customer/portal/articles/928638#targetText=How%20often%20can%20I%20change%20my%20username%3F>`_.
+
+* Information about textures is updated not more often than
+  `once a minute <https://wiki.vg/Mojang_API#UUID_-.3E_Profile_.2B_Skin.2FCape>`_.
+
+If you own a Minecraft premium account, but your nickname is busy, please contact our
```

Comment:
I think this information should also be doubled in an article for players, if it isn't already.

Reply:
It's already done: https://account.ely.by/rules#rule-2-5